rllib/algorithms/tests/test_algorithm.py

```diff
@@ -347,6 +350,30 @@ def test_no_env_but_eval_workers_do_have_env(self):
         bc.train()
         bc.stop()
 
+    def test_eval_workers_on_infinite_episodes(self):
```

Comment:
New test to check for proper behavior if eval workers are set to run by episodes, but episodes dont terminate.